bedrock/settings/__init__.py

```diff
@@ -270,6 +270,7 @@
     # CSP directive updates we're testing that we hope to move to the enforced policy.
     CONTENT_SECURITY_POLICY_REPORT_ONLY["DIRECTIVES"]["frame-ancestors"] = [csp.constants.NONE]
     CONTENT_SECURITY_POLICY_REPORT_ONLY["DIRECTIVES"]["style-src"].remove(csp.constants.UNSAFE_INLINE)
+    CONTENT_SECURITY_POLICY_REPORT_ONLY["DIRECTIVES"]["img-src"].remove("creativecommons.org")
```

Comment:
I think we are pretty safe just to remove `creativecommons.org` from `_csp_img_src` entirely, as this seems to be the only image we load from that domain.